content/en/error_tracking/backend/getting_started.md

```diff
@@ -0,0 +1,439 @@
+---
+title: Backend Error Tracking
+aliases:
+- /error_tracking/standalone_backend/getting_started
+further_reading:
+- link: '/error_tracking/issue_states/'
+  tag: 'Documentation'
+  text: 'Error Tracking Issue States and Workflows'
+- link: '/error_tracking/backend/exception_replay'
+  tag: 'Documentation'
+  text: 'Simplify production debugging with Datadog Exception Replay'
+- link: "/error_tracking/explorer"
+  tag: "Documentation"
+  text: "Learn about the Error Tracking Explorer"
+---
+
+## Overview
+
+[Error Tracking][1] processes errors collected by the Datadog Agent. Whenever an [error span][2] or span containing an error [span event][3] is collected, Error Tracking processes and groups it under an issue, or group of similar errors.
+
+## Getting started
+
+Follow the [in-app setup instructions][4] or choose an instrumentation approach to start collecting backend errors:
+
```

Comment:
We should probably list out and link to the two approaches

- Single Step Instrumentation
- Manual Instrumentation